specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-06-01/netapp.json

```diff
@@ -4128,18 +4124,11 @@
       }
     },
     "snapshotPolicyVolumeList": {
-      "description": "Volumes associated with snapshot policy",
-      "type": "object",
-      "properties": {
-        "value": {
-          "description": "List of volumes",
-          "type": "array",
-          "items": {
-            "type": "object",
-            "readOnly": true,
-            "description": "Associated volume resource Id"
-          }
-        }
+      "description": "List of volumes",
+      "type": "array",
```

Comment:
This is a significant breaking change to what existed before and doesn't meet the RPC defined collection response.  It should be { "value": [{yourObject1}, {yourObject2}] }.

Reply:
Yes, this was to match the actual response from the service as SDK's where having deserialization issues. 
Probably we would need to update the service to produce { "value": [{yourObject1}, {yourObject2}] } type response. Is this a hard requirement and something that what we should do here?

Reply:
updated spec

Reply:
The collection response is a standard contract that everyone should follow for collection GETs.